### ISSUE TITLE
Mismatch in Data Statistics

### ISSUE THREAD

**Issue by hshreeshail** [how2sign/how2sign-data#4]
I found discrepancies between the different sources citing how many data points there are in the train/test/val data splits. The different sources for example are the video folders downloaded from the [website](https://how2sign.github.io/#download), the list in the csv files from the website, and the numbers quoted in Table:2 of the How2Sign CVPR21 paper. I am quoting the different numbers according to these different source down below:

TRAIN SPLIT: 
front_clips = 31048 (i.e. number of .mp4 files in train_rgb_front_clips directory)
side_clips = 31165 (i.e. number of .mp4 files in train_rgb_side_clips directory)
csv_file = 31165 (i.e. number of files listed in how2sign_realigned_val.csv)
table:2 = 31128

TEST SPLIT: 
front_clips = 2343
side_clips = 2357
csv_file = 2357
table:2 = 2322

VAL SPLIT: 
val_rgb_front_clips = 1739
val_rgb_side_clips = 1741
csv_file = 1741
table:2 = 1741

**Comment by hshreeshail** [comment]
typo under TRAIN_SPLIT: csv_file: I meant number of files listed in `how2sign_realigned_train.csv`

**Comment by FangSen9000** [comment]
@hshreeshail 
Yes, there are some videos that are not actually available, which causes a folder or two to be missing. Not just videos, but some texts. For example, there's a missing piece in video number 1550 that looks like. I will address these challenges in my future data sets.
